lib/publiccloud/provider.pm

```diff
@@ -466,6 +466,24 @@ sub terraform_destroy {
     record_info('ERROR', 'Terraform exited with ' . $ret, result => 'fail') if ($ret != 0);
 }
 
+sub terraform_param_tags
+{
+    my ($self) = @_;
+    my $tags = {
+        openqa_ttl        => get_var('MAX_JOB_TIME', 7200) + get_var('PUBLIC_CLOUD_TTL_OFFSET', 300),
+        openqa_var_JOB_ID => get_current_job_id(),
+        map { 'openqa_var_' . $_ => get_var($_) } qw(NAME TEST VERSION FLAVOR DISTRI BUILD),
+    };
+
+    return encode_json($tags);
```

Comment:
```suggestion
    return escape_single_quote(encode_json($tags));
```

Reply:
I don't understand who is "callee" here ? we speaking about function which does not have input params . so can you give me more detailed example which will show that someone passing `"` so it breaks everything :thinking: